src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprints.java

```diff
@@ -47,6 +48,9 @@
 public class DockerFingerprints {
     
     private static final Logger LOGGER = Logger.getLogger(DockerFingerprints.class.getName());
+
+    @SuppressFBWarnings(value="MS_SHOULD_BE_FINAL", justification="mutable for scripts")
+    public static boolean DISABLE = Boolean.getBoolean(DockerFingerprints.class.getName() + ".DISABLE");
```

Comment:
Add `@Restricted(NoExternalUse.class)` and add a static getter method to offer API for downstream plugins which may also need this flag?